### PR TITLE
THIRDEYE-854 : Create a manager for DetectionJobScheduler to enable/disable/adhoc detection jobs

### PR DESCRIPTION
**Pull request by npawar**
Created management functions in DetectionJobScheduler to enable/disable detection functions and run adhoc. Added endpoints to call these manager functions in DetectionJobResource. These endpoints will replace the existing calls made from AnomalyResource CRUD to the AnomalyDetectionJobResource

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/test/java/Test.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.linkedin.thirdeye.constant.MetricAggFunction;
+import com.linkedin.thirdeye.detector.api.AnomalyFunctionSpec;
+
+public class Test {
```
Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/test/java/Test.java]: Can you please convert this to junit test ?

Comment (npawar) [inline thirdeye/thirdeye-pinot/src/test/java/Test.java]: This was a test class for a completely different thing. Sorry forgot to erase if before adding. Removed.

**Comment by puneetjaiswal** [comment]
LGTM

**Comment by npawar** [comment]
Testing done:
Tested the endpoints using curl calls
Adhoc run - curl -i -X POST "http://localhost:1340/api/detection-job/65/ad-hoc"
Enable function - curl -i -X POST "http://localhost:1340/api/detection-job/65"
Disable function - curl -i -X DELETE "http://localhost:1340/api/detection-job/65"

**Pull request merged**